pyproject.toml

```
@@ -0,0 +1,17 @@
+[project]
+name = "pypolyclip"
+description = "A python driver for polyclip written in C by JD Smith."
+dynamic = [
+    "version"
+]
+
+[build-system]
+requires = ["setuptools", "wheel", "numpy"]
```

```suggestion
requires = ["setuptools", 
            "setuptool_scm>=6.2",
            "oldest-supported-numpy"]
build-backend = "setuptools.build_meta"
```

@pllim Should `oldest-supported-numpy` still be used now that `numpy 1.25` is released?

---

@pllim I pinged you here.

---

Ah, thanks! Must have been hidden by all the other comments, so didn't show up in quick search. 

You could drop it but there are some unresolved discussions about ABI (see `seberg` comments) and `#define` over at `astropy`. See if they apply to you or not first: https://github.com/astropy/astropy/pull/14949

---

I suppose we should just use `oldest-supported-numpy` for now.  We can revisit this later once things are settled.